### PR TITLE
Add html-loader, move HTML templates to separate files.

### PR DESCRIPTION
**Pull request by StevePole**

### CODE REVIEW

File: angular/forms/amount-currency-select/tw-amount-currency-select.component.js [hunk 1]
```diff
@@ -32,47 +32,7 @@ const TwAmountCurrencySelect = {
...
-          options="$ctrl.currencies" \
-          filter="{{ $ctrl.currencyFilterPlaceholder }}" \
-          ng-change="$ctrl.changedCurrency()"> \
-            <a href="" ng-if="!!$ctrl.customActionLabel" ng-click="$ctrl.onCustomAction()"> \
-              {{ $ctrl.customActionLabel }} \
-            </a> \
-        </tw-select> \
-      </span> \
-    </div>',
+  template: require('./amount-currency-select.html')
```
Comment (oliverviljamaa) [inline angular/forms/amount-currency-select/tw-amount-currency-select.component.js]: Why not `import template from './amount-currency-select.html';` and then use the object property shorthand?

Comment (StevePole) [reply angular/forms/amount-currency-select/tw-amount-currency-select.component.js]: No particular reason.  I was following a couple examples that did it this way, but your approach is probably better, will change.

Comment (oliverviljamaa) [reply angular/forms/amount-currency-select/tw-amount-currency-select.component.js]: We use the import and shorthand in most of our other Angular repos, it's pretty neat:
https://github.com/transferwise/calculator/blob/master/src/calculator/calculator.component.js

Comment (oliverviljamaa) [reply angular/forms/amount-currency-select/tw-amount-currency-select.component.js]: bitmoji

**Pull request merged**